colossalai/zero/init_ctx/init_context.py

```diff
@@ -126,18 +127,31 @@ def __init__(self,
         self.model_numel_tensor = model_numel_tensor
         self.dp_process_group = dp_process_group or gpc.get_group(ParallelMode.DATA)
 
+    def _pre_context_exec(self):
+        """ 
+        The Callback function when entering the context
+        """
+        self.logger = get_dist_logger("ZeroInitContext")
+        GLOBAL_MODEL_DATA_TRACER.start()
+
     def _post_context_exec(self):
-        """The callback function when the context exits.
+        """The callback function when exiting context.
         """
         if not self.rm_torch_payload_on_the_fly:
             for param in self.initialized_param_list:
                 assert hasattr(param, 'col_attr')
                 param.col_attr.remove_torch_payload()
 
             del self.initialized_param_list
+        GLOBAL_MODEL_DATA_TRACER.close()
+        cuda_mem_MB = GLOBAL_MODEL_DATA_TRACER.cuda_usage / 1e6
+        self.logger.info(f"Existing ZeRO Context Model Data CUDA Memory Usage {cuda_mem_MB} MB", [0])
+        disable_existing_loggers(["ZeroInitContext"])
```

Comment:
Why disable?

Reply:
We do not need a logger under the context of ZeroInitContext

Reply:
Maybe you should use `self.logger.set_level()`. I will modify the API of `disable_existing_loggers` later. It will disable all existing loggers except `["ZeroInitContext"]` now.

Reply:
set level is not what I want. I wish the logging displaying the string ZeroInitContext.

Reply:
remove this line